src/textures/TextureSource.js

```diff
@@ -144,7 +144,7 @@ var TextureSource = new Class({
          * @type {boolean}
          * @since 3.20.0
          */
-        this.isVideo = (source instanceof HTMLVideoElement);
+        this.isVideo = (window.hasOwnProperty('HTMLVideoElement') && source instanceof HTMLVideoElement);
```

Comment:
I wonder if this should be a `customEnvironment` config check instead. Although, checking `window` may be less code perhaps.

Reply:
yes, please consider about a solution, anyway for now this works in our env, other codes related to video gameobject are not compiled into the final output, only this line throws error.